app/manifest/v3/_base.json

```diff
@@ -73,7 +73,8 @@
     "scripting",
     "storage",
     "unlimitedStorage",
-    "webRequest"
+    "webRequest",
+    "offscreen"
```

Comment:
note: this change will trigger a user confirmation of policies upon update

Reply:
Only for the MV3 version though, right?